Week7/CurrentNews/app/src/main/java/com/florencenjeri/currentnews/database/NewsRepository.kt

```diff
@@ -6,19 +6,24 @@ import com.florencenjeri.currentnews.App
 import com.florencenjeri.currentnews.model.News
 import com.florencenjeri.currentnews.model.Success
 import com.florencenjeri.currentnews.network.NetworkStatusChecker
+import com.florencenjeri.currentnews.network.RemoteApi
+import org.koin.core.KoinComponent
+import org.koin.core.inject
 
-class NewsRepository(val dao:NewsDao) {
+class NewsRepository : KoinComponent {
+    private val remoteApi: RemoteApi by inject()
```

Comment:
These should be injected through the construcotr and not Koin.

Only inject in Fragments/Activities!